webdriver/tests/execute_async_script/execute_async.py

```diff
@@ -26,6 +26,82 @@ def test_no_browsing_context(session, closed_window):
     assert_error(response, "no such window")
 
 
+@pytest.mark.parametrize("values", [
+      {"primitive": "null", "expected": None},
+      {"primitive": "undefined", "expected": None},
+      {"primitive": "true", "expected": True},
+      {"primitive": "false", "expected": False},
+      {"primitive": "23", "expected": 23},
+      {"primitive": "'a string'", "expected": "a string"}
+    ])
```

Comment:
nit: Instead of parametrizing on a dictionary with `primitive` and `expected` keys, I feel it would read better if you just exposed both values as arguments:

```python
@pytest.mark.parametrize("primitive,expected", [("null", None), ("undefined", None), ("true", True), …])
def test_primitive_serialization(session, primitive, expected):
    …
```